HW03/Lizaveta_Muromskaya/lizaveta_muromskaya_hw_03_t_03.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+TIME = /[0-9]+:[0-9]+:[0-9]+\.[0-9]/.freeze
+def time_in_sec(time)
+  time.each do |time_numbers|
+    time_buf = time_numbers.split(':')
+    sec = time_buf[0].to_f * 3600 + time_buf[1].to_f * 60 + time_buf[2].to_f
+    @seconds << sec
+  end
+end
+
+def time_difference
+  @seconds.each_cons(2) do |start_event, end_event|
+    res = end_event - start_event
+    @result << res.round(1).to_s if res.positive?
+  end
```

Comment:
Methods shouldn't create any side effects ( like updating global variables etc ). You have 2 options here:
- wrap everything into a class  ( inside the class it's ok )
- just send argument and return value 
```suggestion
def time_difference(seconds)
  seconds.each_cons(2).map do |start_event, end_event|
    res = end_event - start_event
    res.positive? ? res.round(1).to_s : nil
  end.compact
```